contribs/gmf/apps/mobile/index.html.ejs

```diff
@@ -60,7 +60,7 @@
       <button ngeo-mobile-geolocation=""
         ngeo-mobile-geolocation-map="::mainCtrl.map"
         ngeo-mobile-geolocation-options="::mainCtrl.mobileGeolocationOptions">
-        <span class="fa fa-dot-circle-o"></span>
+        <span class="far fa-dot-circle"></span>
```

Comment:
I'm not sure we should have a mix of `fa` and `far` classes

Reply:
@fredj https://fontawesome.com/icons/dot-circle?style=regular The solid one don't match our needs. And we can't use always the "regular" police because the search is not free.